src/components/Blogcontainer.jsx

```diff
@@ -1,76 +1,78 @@
-  var addToList = (e) => {
+  
+  let addToList = (e) => {
     e.preventDefault();
-    setNewPostItem([...newPostItem, input.current.value]);
+    setPosts([...posts, { text: input.current.value, user: "John Doe", likes: [] }]);
+    input.current.value = "";
   };
 
-  const clearInput = () => (input.current.value = "");
-  return (
+   return (
     <>
-      <div className="containerInput">
-        <div className="blogContainer">
+      <div className="flex justify-center  pb-2">
+        <div className="border-purple-900 bg-gray-200 border-solid justify-evenly  rounded-2xl w-4/5	flex  items-center p-3 ">
           <h4>Create post</h4>
-          <div>
-            <form className="blog" onSubmit={clearInput}>
-              <label>
-                <input
-                  ref={input}
-                  type="text"
-                  id="height"
-                  placeholder="Share your thoughts..."
-                />
-              </label>
-              <button onClick={addToList} className="postBtn">
-                Post
-              </button>
-            </form>
-          </div>
+
+          <form className="flex items-center p-3">
+            <label>
+              <input
+                ref={input}
+                type="text"
+                className="rounded-2xl leading-10 w-96 outline-none text-md"
+                 placeholder="    Share your thoughts..."
+              />
+            </label>
+
+            <button
+              onClick={addToList}
+              className="leading-8 ml-4 rounded-2xl  bg-gray-400 w-20  border-none relative text-base hover:text-white	"
+            >
+              Share
+            </button>
+          </form>
         </div>
       </div>
 
-      {newPostItem.map((item, index) => (
-        <div className="postBottomLeft" key={index}>
-          <div className="new_post">
-            <div className="List">
-              <div className="postTop">
-                <div className="postTopLeft">
-                  <img
-                    className="postProfileImg"
-                    src={Users[0].profilePicture}
-                    alt=""
-                  />
-                  <p className="postUsername">{Users[0].username}</p>
-                </div>
-              </div>{" "}
-              <p className="inputList">{item} </p>
+       {posts.map((item, index) => (
+        <div key={index} className="flex justify-center p-2.5">
+        <div className="flex justify-center p-4 items-center overflow-auto break-words	pl-3  w-10/12 bg-gray-200 rounded-3xl   border-solid " >
+    
+           
+              <img
+                className="h-20   rounded-full  "
+                src={context[0].profilePicture}
+                alt=""
+              />
+              <p className="text-base p-2 flex italic w-20">
+                <strong>{context[0].username}</strong>
+              </p>
+        
+
+            <p className="italic break-words ml-4 w-4/6 ">{item} </p>
```

Comment:
this is what is causing the error. You should say `{item.text}` because item now is a post object